haversine-processing-engine/package-lock.json

```diff
@@ -0,0 +1,1742 @@
+{
```

Comment:
why commit the package-lock.json file? Think this file should be in the .gitignore list.